### PR TITLE
Added documentation for few shot prompting

### LINKED ISSUE #3110: Add docs for `FewShotChatMessagePromptTemplate`

### PR DESCRIPTION
**Pull request by bracesproul**
Closes #3110

### CODE REVIEW

File: docs/docs/modules/model_io/prompts/prompt_templates/few_shot.mdx [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+```typescript
+import { ChatOpenAI } from "langchain/chat_models/openai";
+```
+
+```typescript
+const model = new ChatOpenAI({});
+
+const prompt =
+  ChatPromptTemplate.fromTemplate(`Rephrase the users query to be more general, using the following examples:
```
Comment (jacoblee93) [inline docs/docs/modules/model_io/prompts/prompt_templates/few_shot.mdx:93]: We should use `prefix` on `ChatPromptTemplate` for this no?

**Review by jacoblee93 — COMMENTED** [review]
x

File: docs/docs/modules/model_io/prompts/prompt_templates/few_shot.mdx (hunk 1 above)
Comment (bracesproul) [reply docs/docs/modules/model_io/prompts/prompt_templates/few_shot.mdx:93]: Yes, good call

File: docs/docs/modules/model_io/prompts/prompt_templates/few_shot.mdx [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+```typescript
+const formattedPrompt = await fewShotPrompt.format({});
+console.log(formattedPrompt);
+```
+
+```typescript
+[
+  HumanMessage {
+    lc_namespace: [ 'langchain', 'schema' ],
+    content: 'Human: Could the members of The Police perform lawful arrests?\n' +
```
Comment (jacoblee93) [inline docs/docs/modules/model_io/prompts/prompt_templates/few_shot.mdx:70]: I thought the chat version of this returned human + AI message pairs?

Comment (bracesproul) [reply docs/docs/modules/model_io/prompts/prompt_templates/few_shot.mdx:70]: Only if you input human & ai. If you just give it human it'll only return human and viceversa

**Pull request merged**